wpeverest/user-registration
# UR-181 Feature - Autocomplete Address Field with Map

### All Submissions:






### Changes proposed in this Pull Request:

This PR is to make geolocation compatible with new feature i.e Autocomplete Address Field with Map. 

### How to test the changes in this Pull Request:

1. Goto Settings->Geolocation there enter Google Map API key and enable current location if you want.
2. Goto to Form Build Drag Input field in field option you can see option to enable autocomplete address field. 
3. When you enable you can see next option i.e address style which gives you option to choose only address or with map.
4. Goto registration page and edit profile page and see whether that input field is working properly or not.

### Types of changes:




### Other information:




### Changelog entry

> Feature - Autocomplete Address Field with Map.
